docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -22,39 +32,74 @@ This guide will talk through how to set up Teleport with Slack. Teleport's Slack
   Your browser does not support the video tag.
 </video>
 
-## Setup
+## Prerequisites
 
-### Prerequisites
+- Slack admin privileges to create an app and install it to your workspace
 
-This guide assumes that you have:
+(!/docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
-- A running Teleport Cluster
-- Admin privileges with access to `tctl`
-- Slack admin privileges to create an app and install it to your workspace
+## Step 1/7. Install the Teleport Slack plugin
+
+We currently only provide `linux-amd64` binaries. You can also compile these plugins
+from source.
+
+<Notice scope={["oss","enterprise"]} type="tip">
```

Comment:
The first notice of this guide mentions we only support enterprise version
But here, we have `oss` as part of the scope
Should we remove `oss` here?

Reply:
That was a mistake—removed.